synapse/lib/cell.py

```diff
@@ -467,25 +479,34 @@ class Cell(s_nexus.Pusher, s_telepath.Aware):
             'description': 'Record all changes to the cell.  Required for mirroring (on both sides).',
             'type': 'boolean'
         },
+        'dmon:listen': {
+            'description': 'A config-driven way to specify the telepath bind URL.',
+            'type': ['string', 'null'],
+        },
+        'https:port': {
+            'description': 'A config-driven way to specify the HTTPS port.',
+            'type': ['integer', 'null'],
+        },
     }
 
-    async def __anit__(self, dirn, conf=None, readonly=False, *args, **kwargs):
+    async def __anit__(self, dirn, conf=None, readonly=False):
```

Comment:
I'd suggest we leave the args/kwargs in. We can't just be changing call signatures on a whim in a regressive fashion.

Reply:
Nothing currently uses this anti-pattern and we want to keep it that way.